src/helpers/helpers.canvas.js

```diff
@@ -80,6 +93,8 @@ export function drawBox(ctx, rect, options) {
   ctx.closePath();
   ctx.fill();
   if (stroke) {
+    // sets shadow color for border
```

Comment:
these comments were probably more meaningful before changing to background/borderShadowColor, now they are quite useless can be removed (all instances)

Reply:
:+1:

Reply:
done